packages/block-library/src/image/edit.js

```diff
@@ -675,12 +675,10 @@ class ImageEdit extends Component {
 								<>
 									{ getInspectorControls( imageWidth, imageHeight ) }
 									<ResizableBox
-										size={
-											width && height ? {
```

Comment:
Do you know why the check was there?

Reply:
The logic of just applying sizes when both dimension change was applied in this PR https://github.com/WordPress/gutenberg/pull/6205. I did not notice regressions on the behavior that PR was intended to solve when comparing this PR to master (before the merge).